recipes/pycgtool/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pycgtool" %}
+{% set version = "2.0.0b3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycgtool-{{ version }}.tar.gz
+  sha256: 7f70c55fc0c00ddf54a6d1701e51b9b42aa930edbcbf5b2e4cfc642d577bf2ab
+
+build:
+  number: 0
+  skip: true   # [py>=40 or py2k]
+  entry_points:
+    - pycgtool = pycgtool.__main__:main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - poetry
+  run:
+    - astunparse ==1.6.2
+    - cython >=0.29.21,<0.30.0
```

Comment:
Is cython needed at run time? Usually it is only a host/build dependency.

Reply:
Hmm, no, on second thought it actually shouldn't be required at all for conda. It's a requirement of the `pip` version in order to satisfy one of the other dependencies, but that dependency is already built in this case

Reply:
Even when pip installing this is a workaround. Ideally the dependency should declare cython as its build dependency, or better yet, build and publish wheels. It is not a big deal for your to supply cython to make things work but it is not encouraged as a practice. I know... Python packaging is messy...